### DISCUSSION TITLE
uTLS 指纹一问

### DISCUSSION THREAD

**Discussion by kingwilliam** [XTLS/Xray-core#1694] [General]
我对国内网络环境不是太了解, 最近谈得好火的uTLS fingerprint其中一个建议选项是用 chrome 指纹. 但正常情况下, 国内不是不应出现chrome吗?

**Comment by qianxuu** [comment]
chrome 有可直接访问/下载的[中国官网](https://google.cn/chrome/)，位于 google.cn 域名下，另外 chrome 在国内很流行，他有个中文俗称叫谷歌浏览器，随处可见，并不奇怪，即使是政府网站也会提到请使用谷歌浏览器访问

**Comment by kingwilliam** [comment]
明白 原来是这样 谢谢解答

**Reply by Smallthing** [reply]
好奇你在哪里- -？平行宇宙？

**Reply by kingwilliam** [reply]
香港
